packages/test-generator/integration-test-templates/cypress/e2e/create-form-spec.cy.ts

```diff
@@ -30,7 +30,7 @@ describe('CreateForms', () => {
         customValidation: 'All dog emails are yahoo emails',
         ip: 'The value must be an IPv4 or IPv6 address',
       };
-      cy.get('#customFormCreateDog').within(() => {
+      cy.get('#customFormCreateDog', { timeout: 8000 }).within(() => {
```

Comment:
The data initialization takes a bit more time to finish now that we have more to do